app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainFragment.kt

```diff
@@ -50,7 +50,7 @@ class MainFragment : Fragment() {
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
-        viewModel.stock.observe(viewLifecycleOwner, Observer { stocks ->
+        viewModel.stocks.observe(viewLifecycleOwner, Observer { stocks ->
```

Comment:
I changed 'stock' to 'stocks' because the variable is getting a list of multiple stock data. The importance of good variable name was discussed in this article : https://medium.com/jay-tillu/variables-in-kotlin-d295cbf3b3eb.